requirements.txt

```diff
@@ -85,28 +81,28 @@ pycparser==2.19
 pygments==2.3.1
 pyjwt==1.7.1
 pyphen==0.9.5
-python-dateutil==2.8.0 ; python_version >= '2.7'
-python3-openid==3.1.0 ; python_version >= '3.0'
+python-dateutil==2.8.0
+python3-openid==3.1.0
 pytz==2018.9
 raven==6.9.0
 razorpay==1.2.0
 redis==2.10.6
-requests-oauthlib==1.2.0
 requests==2.21.0
+requests-oauthlib==1.2.0
 rx==1.6.1
 s3transfer==0.2.0
 singledispatch==3.4.0.3
-six==1.12.0
+six==1.11.0
 social-auth-app-django==3.1.0
 social-auth-core==3.1.0
 sqlparse==0.3.0
-stripe==2.21.0
+stripe==2.23.0
 sympy==1.3
 text-unidecode==1.2
-tinycss2==1.0.1
+tinycss2==1.0.2
 typing==3.6.6
-urllib3==1.24.1
-uwsgi==2.0.18 ; platform_system != 'Windows'
-vine==1.2.0
+urllib3==1.23
+uwsgi==2.0.18
```

Comment:
Poetry doesn't support export the markers such as ```; platform_system != 'Windows'```?

Reply:
Good catch. I opened an issue on poetry: https://github.com/sdispater/poetry/issues/1011.

Reply:
This was also added. Just needs a release.